README.md

```diff
@@ -9,3 +9,27 @@
 
 
 A simple Python CLI script to send emails from gmail.
+
+Installation steps:
+
+. First set up your gmail account to allow access
+> In your google account page- https://myaccount.google.com/ , go to Security Tab and allow less secure app access
+
+> If you don't want to compromise with security , use this link to allow access using OAuth2 authorization framework - https://developers.google.com/gmail/api/quickstart/python
+
+. Clone the repository or download the zip and extract
+
+. Navigate to the folder and run the following commands
+       
+       python3 setup.py build
+       
+       python3 setup.py install
```

Comment:
After repo cloning, the preferred installation is by running `pip install -e .`
Otherwise the following error would be produced.

![image](https://user-images.githubusercontent.com/48885301/94951068-48d5a800-0501-11eb-8d7e-c8b3fec9bf0b.png)

Reply:
readme has been updated